scripts/issue/23/scrape_tokens.py

```diff
@@ -0,0 +1,39 @@
+from brownie import config
+import requests
+import pandas as pd
+
+def main():
+    """
+    Get all ERC-20 tokens and their balances in a csv for a given address with ethplorer api
+    
+    `scrape` can be called directly from command line with an address:
+    brownie run scrape_tokens.py scrape <address>
```

Comment:
since this is a nice tool we will reuse, i suggest to create a folder such as `scripts/gnosis/` and place it there. it will be very difficult to find again in `scripts/issue/.../` in a couple of weeks/months.